pkg/kubernetes/config.js

```diff
@@ -117,7 +117,37 @@ define([
         return scheme;
     }
 
+    function versionCompare(v1, v2) {
```

Comment:
I'll change this to use the standard underscore syntax we have for functions in this file.